<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteControlSettingsDialog</class>
 <widget class="QDialog" name="RemoteControlSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Remote Control Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="devicesTab">
           <attribute name="title">
            <string>Devices</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableWidget" name="devices">
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <column>
               <property name="text">
                <string>Label</string>
               </property>
               <property name="toolTip">
                <string>Label used for the device in the GUI</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Device</string>
               </property>
               <property name="toolTip">
                <string>Device name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Model</string>
               </property>
               <property name="toolTip">
                <string>Device model</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Protocol</string>
               </property>
               <property name="toolTip">
                <string>Protocol used to communicate with the device</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="add">
                <property name="text">
                 <string>Add...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="remove">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="edit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Edit...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deviceUp">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>Move up</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../sdrgui/resources/res.qrc">
                  <normaloff>:/arrow_up.png</normaloff>:/arrow_up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deviceDown">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>Move down</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../sdrgui/resources/res.qrc">
                  <normaloff>:/arrow_down.png</normaloff>:/arrow_down.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settingsTab">
           <attribute name="title">
            <string>Settings</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="tpLinkGroup">
              <property name="title">
               <string>TP-Link</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="tpLinkUsernameLabel">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="tpLinkUsername">
                 <property name="toolTip">
                  <string>Username. Typically your email address</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="tpLinkPasswordLabel">
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="tpLinkPassword">
                 <property name="toolTip">
                  <string>Password</string>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::PasswordEchoOnEdit</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="homeAssistantGroup">
              <property name="title">
               <string>Home Assistant</string>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="homeAssistantTokenLabel">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Access Token</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="homeAssistantHostLabel">
                 <property name="text">
                  <string>Host</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="homeAssistantToken">
                 <property name="toolTip">
                  <string>API access token. Can be generated on your profile page: http://homeassistant.local:8123/profile</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="homeAssistantHost">
                 <property name="toolTip">
                  <string>Hostname of computer running Home Assistant. Typically http://homeassistant.local:8123/profile</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="visaGroup">
              <property name="title">
               <string>VISA</string>
              </property>
              <layout class="QFormLayout" name="formLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="visaResourceFilterLabel">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Resource filter</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="visaLogIOLabel">
                 <property name="text">
                  <string>Log IO</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="visaResourceFilter">
                 <property name="toolTip">
                  <string>Regular expression of VISA resources not to connect to</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="visaLogIO">
                 <property name="toolTip">
                  <string>When checked, VISA input and output is written to SDRangel log</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="deviceGroup">
              <property name="title">
               <string>Devices</string>
              </property>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="updatePeriodLabel">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Update period (s)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="updatePeriod">
                 <property name="toolTip">
                  <string>Period in seconds between requests to update device state</string>
                 </property>
                 <property name="minimum">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>1.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="chartsGroup">
              <property name="title">
               <string>Charts</string>
              </property>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="chartVerticalPolicyLabel">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Height</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="chartVerticalPolicy">
                 <item>
                  <property name="text">
                   <string>Expanding</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Fixed</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="chartHeightLabel">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Height (pixels)</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="chartHeight">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <number>50</number>
                 </property>
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                 <property name="value">
                  <number>130</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RemoteControlSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RemoteControlSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
